src/test/java/com/github/tomakehurst/wiremock/stubbing/JsonTest.java

```diff
@@ -110,6 +113,131 @@ public void counts1ForEmptyObject() {
+            + "    \"carriage return : \\r padding\",\n"
+            + "    \"horizontal tab: \\t padding\",\n"
+            + "    \"hex digit: \\u12ab padding\"\n"
+            + "  ]\n"
+            + "}";
+    JsonNode jsonNode = Json.node(json);
+    String result = Json.toStringEscaped(jsonNode);
+    String jsonCompressedAndEscaped =
+        jsonNode
+            .toString()
+            .replace("\\\"", "\"")
+            .replace("\\\\", "\\")
+            .replace("\\b", "\b")
+            .replace("\\f", "\f")
+            .replace("\\n", "\n")
+            .replace("\\r", "\r")
+            .replace("\\t", "\t")
+            .replaceAll("\\\\u12ab", "\u12ab");
+    assertThat(result, is(jsonCompressedAndEscaped));
+  }
+
+  @Test
+  public void testToByteArrayEscaped() {
+    // language=JSON
+    String json =
+        "{\n"
+            + "  \"string\": \"This is a text\",\n"
+            + "  \"number\": 1,\n"
+            + "  \"boolean\": true,\n"
+            + "  \"null\": null,\n"
+            + "  \"simple_array\": [\n"
+            + "    \"element1\",\n"
+            + "    \"element2\",\n"
+            + "    \"element3\"\n"
+            + "  ],\n"
+            + "  \"object\": {\n"
+            + "    \"children_string\": \"This is a text\",\n"
+            + "    \"children_number\": 1\n"
+            + "  },\n"
+            + "  \"object_array\": [\n"
+            + "    {\n"
+            + "      \"id\": 1,\n"
+            + "      \"name\": \"one\"\n"
+            + "    },\n"
+            + "    {\n"
+            + "      \"id\": 2,\n"
+            + "      \"name\": \"two\"\n"
+            + "    },\n"
+            + "    {\n"
+            + "      \"id\": 3,\n"
+            + "      \"name\": \"three\"\n"
+            + "    }\n"
+            + "  ],\n"
+            + "  \"escape_sequence\":[\n"
+            + "    \"quotation mark : \\\" padding\",\n"
+            + "    \"reverse solidas : \\\\ padding\",\n"
+            + "    \"backspace : \\b padding\",\n"
+            + "    \"formfeed : \\f padding\",\n"
+            + "    \"newline : \\n padding\",\n"
+            + "    \"carriage return : \\r padding\",\n"
+            + "    \"horizontal tab: \\t padding\",\n"
+            + "    \"hex digit: \\u12ab padding\"\n"
+            + "  ]\n"
+            + "}";
+    JsonNode jsonNode = Json.node(json);
+    byte[] result = Json.toByteArrayEscaped(jsonNode);
+    byte[] jsonCompressedAndEscaped =
+        jsonNode
+            .toString()
+            .replace("\\\"", "\"")
+            .replace("\\\\", "\\")
+            .replace("\\b", "\b")
+            .replace("\\f", "\f")
+            .replace("\\n", "\n")
+            .replace("\\r", "\r")
+            .replace("\\t", "\t")
+            .replaceAll("\\\\u12ab", "\u12ab")
+            .getBytes(StandardCharsets.UTF_8);
+    assertThat(result, is(jsonCompressedAndEscaped));
+  }
+
+  @Test
+  public void test2() {
```

Comment:
Super small minor thing but I was wondering if we could add a more targetted name for this test?

Reply:
This test is not really needed. I had it for my local testing. 😅